archlinux/PKGBUILD

```diff
@@ -67,22 +66,20 @@ package_qubes-vm-core() {
     optdepends=(gnome-keyring gnome-settings-daemon python-nautilus gpk-update-viewer qubes-vm-networking qubes-vm-keyring)
     install=PKGBUILD.install
 
-    # Note: Archlinux removed use of directory such as /sbin /bin /usr/sbin (https://mailman.archlinux.org/pipermail/arch-dev-public/2012-March/022625.html)
-    # shellcheck disable=SC2154
-    make -C qrexec install DESTDIR="$pkgdir" SBINDIR=/usr/bin LIBDIR=/usr/lib SYSLIBDIR=/usr/lib
-
     make install-corevm DESTDIR="$pkgdir" SBINDIR=/usr/bin LIBDIR=/usr/lib SYSLIBDIR=/usr/lib SYSTEM_DROPIN_DIR=/usr/lib/systemd/system USER_DROPIN_DIR=/usr/lib/systemd/user DIST=archlinux
+    make -C app-menu install DESTDIR="$pkgdir" install LIBDIR=/usr/lib SYSLIBDIR=/usr/lib
+    make -C misc install DESTDIR="$pkgdir" install LIBDIR=/usr/lib SYSLIBDIR=/usr/lib
     make -C qubes-rpc DESTDIR="$pkgdir" install
     make -C qubes-rpc/kde DESTDIR="$pkgdir" install
     make -C qubes-rpc/nautilus DESTDIR="$pkgdir" install
     make -C qubes-rpc/thunar DESTDIR="$pkgdir" install
 
     # Remove things non wanted in archlinux
-    rm -r "$pkgdir/etc/yum"*
-    rm -r "$pkgdir/etc/dnf"*
-    rm -r "$pkgdir/etc/init.d"
+    rm -rf "$pkgdir/etc/yum"*
+    rm -rf "$pkgdir/etc/dnf"*
+    rm -rf "$pkgdir/etc/init.d"
```

Comment:
Are those installed at all? Since Makefile rework I think they aren't and those rm can be dropped.

Reply:
Yes I've dropped it. I need to remove that too in gentoo ebuild.